### ISSUE TITLE
Question about precedence, particularly for recipient count (not message size)

### ISSUE THREAD

**Issue by JeremyTBradshaw** [MicrosoftDocs/OfficeDocs-Exchange#1481]
I'm wondering if it's possible to clarify in the **Order of Precedence** section if the exemption that is covered (messages between authenticated senders and recipients) also applies to recipient counts, or if that is only for message size.

### Example

If I have RecipientLimits:Unlimited on my mailbox, and the Receive Connector that will receive my message is set to 200 for MaxRecipientsPerMessage - what happens when:

- I email &gt;200 internal users where they and I are both "authenticated"?
- Same thing but some of those &gt;200 users are internal and others are external?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9a512e6f-47ab-f785-a792-ef82a552c50f
* Version Independent ID: 16869db1-342d-13f2-ce24-0e13a49ec68d
* Content: [Message size limits in Exchange Server](https://docs.microsoft.com/en-us/exchange/mail-flow/message-size-limits?view=exchserver-2019)
* Content Source: [Exchange/ExchangeServer/mail-flow/message-size-limits.md](https://github.com/MicrosoftDocs/OfficeDocs-Exchange/blob/public/Exchange/ExchangeServer/mail-flow/message-size-limits.md)
* Product: **exchange-server-it-pro**
* Technology: ****
* GitHub Login: @msdmaguire
* Microsoft Alias: **dmaguire**

**Comment by JeremyTBradshaw** [comment]
Just wondering if anyone caught wind of this issue yet?

**Comment by JeremyTBradshaw** [comment]
Yeah.. re-reading now, months later, it's still confusing me.  Too much interchange of the terms "Message Size Limits" and "Recipient Limits".  I want to say that "Recipient Limits" should be reserved for use with the topic of recipient counts per message.  "Message Size Limits" should be reserved for use with message / data size topics.

**Comment by j0rt3g4** [comment]
Hello @JeremyTBradshaw 
Let me try to clear your doubts.

## Taking your example:
If I have RecipientLimits:Unlimited on my mailbox and the Receive Connector that will receive my message is set to 200 for MaxRecipientsPerMessage - what happens when:

### I email >200 internal users where they and I are both "authenticated"?
* Let's assume 300 in total internal users

The message is accepted for the first 200 recipients. 
Most messaging servers will continue to resend the message in groups of 200 recipients until the message is delivered to all recipients.

- So in conclusion, it will accept 200 1st and then send them out for the rest 100.

### Same thing but some of those >200 users are internal and others are external?
- The same conclusion takes the 1st 200 no matter if they're internal or externals and then send the other 100.

**Comment by scanum** [comment]
@JeremyTBradshaw Thank you for submitting feedback. We understand that this issue has been resolved.
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Comment by JeremyTBradshaw** [comment]
Thanks @j0rt3g4 , appreciate the descriptive response.

So then all rules of precedence apply to both message size limits and recipient count limits, and so do the exemptions for internal/authenticated users.

Many thanks.

**Comment by mattpennathe3rd** [comment]
@JeremyTBradshaw Just to follow up, I'm investigating some changes to this article to see if I can bring some more clarity. I will report back soon.

**Comment by mattpennathe3rd** [comment]
@JeremyTBradshaw Thanks for waiting! I worked with one of our engineers to get more technical details on this. There will be a new section added about interaction between various limits as well as an example scenario that should be informative.

The updated article will be published in the near future (likely within a few hours of whenever this issue is closed), so when you have time, please feel free to review the additions and offer feedback in a new issue if it does not cover what you need.

Thanks for all of your input in this repo!
